src/workloads/query/multiplanner/Simple.yml

```diff
@@ -0,0 +1,419 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  The goal of this test is to exercise multiplanning. We create as many indexes as possible, and run a
+  query that makes all of them eligible, so we get as many competing plans as possible.
+
+  The original goal of this test was to demonstrate weaknesses of the SBE multiplanner when compared to
+  the classic multiplanner. Mainly, the SBE multiplanner can't round-robin between plans, which means it
+  has to run the list of plans sequentially, which means we can't short-circuit when the shortest-running
+  plan finishes.
+
+  We expect classic to have better latency and throughput than SBE on this workload,
+  and we expect the combination of classic planner + SBE execution (PM-3591) to perform about
+  as well as classic.
+
+GlobalDefaults:
+  dbname: &db test
+  # Collection name used for queries.
+  # Note the 'Loader' actor also hardcodes this name, so it's not actually free to change.
+  coll: &coll Collection0
+
+  docCount: &docCount 1e5
+  resultCount: &resultCount 101
+  selectivity: &selectivity {^NumExpr: {
+    withExpression: "resultCount / docCount",
+    andValues: {resultCount: *resultCount, docCount: *docCount}}}
+
+  maxPhase: &maxPhase 7
+  nop: &Nop {Nop: true}
+
+Actors:
+- Name: DropCollection
+  Type: RunCommand
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [0]
+      NopInPhasesUpTo: *maxPhase
+      PhaseConfig:
+        Repeat: 1
+        Database: *db
+        Operations:
+        - OperationName: RunCommand
+          OperationCommand:
+            drop: Collection0
+
+- Name: SetQueryKnobs
```

Comment:
I think this actor will be identical in the other scenarios, so it can be pulled out into a separate file.